src/components/Root.js

```diff
@@ -41,6 +42,7 @@ const Root = ({ store }) => (
         <PrivateRoute path="/dashboard" component={PageDashboard} />
         <PrivateRoute path="/dirs" component={PageFolders} />
         <PrivateRoute path="/get-started" component={PageGetStarted} />
+        <PrivateRoute path="/InConstruction" component={InConstruction} />
```

Comment:
build all the needed routes like /templates and /requests and load your componenent